### PR TITLE
chore(ci): building and starting local server from dist folder for ci

### LINKED ISSUE #6: Deploy to GitHub Pages
Result should be the same as [`demo-viewer`](https://github.com/voorhoede/demo-viewer). So `examples/` should end up on `https://voorhoede.github.io/demo-linker/examples/` etc.

The [travis-deploy-to-gh-pages](https://github.com/voorhoede/front-end-tooling-recipes/tree/master/travis-deploy-to-gh-pages) recipe can be used for this.

### PR DESCRIPTION
**Pull request by jpsc**
closes #6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,12 @@
   "version": "0.0.0-semantic-releasing",
   "description": "Annotate UI components and link them to wherever you want.",
   "scripts": {
```
Comment (jbmoelker) [inline package.json:5]: Change `"build:dist"` to `"prebuild": "mkdir dist/"` and maybe remove `dist/` first to ensure you always get a clean build?

File: package.json [hunk 2]
```diff
@@ -3,7 +3,12 @@
...
-    "start": "http-server './' -c-1 -o -p 33665",
+    "build": "run-s build:*",
+    "build:dist": "mkdirp dist/",
+    "build:examples": "ncp ./examples/ ./dist/examples/",
+    "build:js": "uglifyjs demo-linker.js -o ./dist/examples/demo-linker.js",
+    "build:css": "csso demo-linker.css --output ./dist/examples/demo-linker.css",
+    "start": "npm run build:examples && http-server './dist/examples/' -c-1 -o -p 33665",
```
Comment (jbmoelker) [inline package.json]: Move `npm run build:examples` to `prestart`?

File: package.json [hunk 3]
```diff
@@ -25,8 +30,13 @@
   "homepage": "https://github.com/voorhoede/demo-linker#readme",
   "devDependencies": {
     "commitizen": "2.8.6",
+    "csso": "2.2.1",
```
Comment (jbmoelker) [inline package.json]: The demo-linker is mainly for development, not so much for production. Do you feel we actually need to minify CSS & JS. If so, shall we copy the original to `dist/demo-linker.(css|js)` and add minified versions `dist/demo-linker.min.(css|js)`? I would then even like it if they had sourcemaps. But maybe we should move this to a separate PR, and focus on getting this deployed to GH Pages first.

**Review by jbmoelker — COMMENTED** [review]
Nice work! Same as with fastatic, I'd like to keep the examples in a separate directory in dist. Mainly because that's still what they are, "examples". And I think the URL should express that. Less so because it prevents any future naming conflicts.

So I would say the resulting dist folder should look like:

```
dist/
  demo-linker.css
  demo-linker.js
  index.html
  examples/
   bootstrap-v3/
   ...
```

The content of the `index.html` file is also more than just a list of examples. So that's why I think it should not be in the examples/ dir.

**Comment by jpsc** [comment]
Updated. I chose to create copy tasks and add them to build.
Also added the script for gh-pages.

**Review by jbmoelker — APPROVED** [review]
